.github/workflows/e2e-browser.yml

```
@@ -14,9 +16,14 @@ jobs:
     runs-on: ubuntu-latest
     environment:
       name: ${{ matrix.environment-name }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         environment-name: ["ESS PodSpaces"]
+        experimental: [false]
```

```suggestion
        experimental: false
```

i don't think you need the brackets here

---

That's also true for the environment name actually

---

so `environmentName` should have it, as there should be multiple there `environmentName: [ "ESS PodSpaces", "ESS Dev-Next" ]`